http.redpanda.sasl.scram/stack.yml

```diff
@@ -9,10 +9,10 @@ services:
   zilla:
     image: "ghcr.io/aklivity/zilla:latest"
     hostname: "zilla"
-    command: [ "start", "-v", "-e" ]
+    command: [ "start", "-v", "-e", "-c", "file:/etc/zilla/zilla.json" ]
```

Comment:
There should be no need to specify `-c` argument in each example.

Please update `ZillaStartCommand` to pull default config location from `.zilla/zilla.properties` so that it can be specified as `/etc/zilla/zilla.json` in the image and work by default instead.